packages/x-components/src/x-modules/search/store/__tests__/actions.spec.ts

```diff
@@ -337,9 +337,6 @@ describe('testing search module actions', () => {
       await store.dispatch('saveOrigin', { location: 'predictive_layer' });
       expect(store.state.origin).toBeNull();
 
-      await store.dispatch('saveOrigin', { feature: 'search_box' });
```

Comment:
Same here.

Reply:
I removed it because right now It makes no sense because you can create an origin with only the feature and this test is to check when it's null.